smartseq2_single_sample/ss2_single_sample.wdl

```diff
@@ -135,9 +139,9 @@ workflow Ss2RsemSingleSample {
   File ref_fasta
   File rrna_interval
   File ref_flat
-  String star_genome
+  File star_genome
```

Comment:
hm, can not recall why I used `String` instead of `File` here....looks like wdl can recognize a a `String` as a `File` if the `String` is pointed to an actually file?

Reply:
Well cromwell/JES does in this case, but dxWDL/DNAnexus doesn't. There are cases where cromwell doesn't do the cast when I thought it would have, so I think the WDL type coercions tend to behave ambiguously. It's probably better to avoid relying on it unless we really need to.